app/src/main/res/layout/home_fragmment.xml

```diff
@@ -25,7 +25,7 @@
     <ListView
         android:id="@+id/homeList"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="fill_parent"
```

Comment:
I meant why couldnt it be a set size all the time that doesnt encompass the entire parent? Why not just set it to 400dp or something?